### ISSUE TITLE
Changes in how Youtube handles automatic captions

### ISSUE THREAD

**Issue by AlifianK** [mpv-player/mpv#13461]
Youtube used to lump automatic captions together with provided subtitles. Now it's under the `automatic_captions` in the video's json metadata, making mpv unable to retrieve the auto captions.

- mpv version v0.37.0-271-gf5c4f0b0

[https://0x0.st/HdeQ.json](https://0x0.st/HdeQ.json)

**Comment by christoph-heinrich** [comment]
I don't think we should automatically add all automatic captions as subtitle tracks, but if you want everything to be listed, you could use this in your config.
```ini
ytdl-raw-options-append=write-subs=
ytdl-raw-options-append=write-auto-subs=
ytdl-raw-options-append=sub-langs=.*
```

**Comment by AlifianK** [comment]
I'm not suggesting we should include all of them but can't the correct auto captions be inferred from `slang` config? For example, `slang = 'eng,en,enUS'` will load the English auto caption? I'm just wondering since mpv used to load auto captions before, if this is the intended behavior from now on then I'll close the issue.

**Comment by christoph-heinrich** [comment]
```
ytdl-raw-options-append=write-subs=
ytdl-raw-options-append=write-auto-subs=
ytdl-raw-options-append=sub-langs=en.*
```
works for me
